contributor_docs/project_wrapups/shantanuKaushik_gsoc_2021.md

```diff
@@ -0,0 +1,57 @@
+  # Adding to p5.js Friendly Error System
+
+#### By Shantanu Kaushik ([@Aloneduckling](https://github.com/Aloneduckling)) | GSoC 2021
+
+### Overview
+Throughout this summer, I worked on p5.js's Friendly Error System (FES in short) under the mentorship of Thales Grilo and Luis Morales-Navarro. The FES is a p5.js feature
+which helps beginners with common errors, transforming error messages into a simple form, and checking for common mistakes. 
+This project aimed to further improve FES by adding new features and adding more to an existing feature.
```

Comment:
This project aimed to further improve FES by further building on an existing feature and adding new features.